core/storage/skill/gae_models.py

```diff
@@ -60,6 +60,10 @@ class SkillModel(base_models.VersionedModel):
     skill_contents = ndb.JsonProperty(indexed=False)
     # The id to be used by the next misconception added.
     next_misconception_id = ndb.IntegerProperty(required=True, indexed=False)
+    # The id of the merged skill, in case the skill has been deduped/merged.
+    superseding_skill_id = ndb.StringProperty(indexed=True, required=False)
```

Comment:
Here and below, note that "required=False" is optional.

Reply:
Not done, but also optional I guess :)

Reply:
Sorry about that, forgot to check in this change